File: python/7 kyu/409.py
# https://www.codewars.com/kata/576bb3c4b1abc497ec000065/train/python

# Passed

def compare(s1, s2):
    def calculate_score(text):
        score = 0
        if not text:
            return score

        for char in text:
            if not char.isalpha():
                return 0
            
            score += ord(char.upper())
            
        return score

    result = calculate_score(s1) == calculate_score(s2)
    return result

output = compare("AD", "BC")
print(output)

File: nodejs/7 kyu/409.js
// https://www.codewars.com/kata/65127141a5de2b1dcb40927e/train/javascript

// Passed

function spinAround(turns) {
  let turnValues = turns.map((turn) => (turn === "right" ? 90 : -90));
  let absTurnInDegrees = Math.abs(
    turnValues.reduce((prev, curr) => prev + curr, 0),
  );
  let result = Math.floor(absTurnInDegrees / 360);
  return result;
}

const output = spinAround([
  "right",
  "right",
  "right",
  "left",
  "right",
  "right",
]);
console.log(output);